2018-05-Berlin/attendee-rsvp.md

```diff
@@ -0,0 +1,127 @@
+# Node.js Collaborator Summit, 2018-05-31 - 2018-06-01, Berlin, Germany
+
+This document serves as an RSVP list for attendees in person, attendees remote,
+and the collaborator's dinner hosted by the Node Foundation.
+
+This is how we will track amount of food and seats. Submit a PR if you need to
+RSVP or update your RSVP.
+
+__PLEASE ADD YOURSELF ALPHABETICALLY :D__
+
+### Attendees In-Person
+
+If you will be in Berlin to attend the Collaborator's Summit - add your name.
+
+1. [Abraham Agiri Jr.](https://github.com/codeekage)
+1. [Adam Miller](https://github.com/amiller-gh)
+1. [Addison Berry](https://github.com/add1sun)
+1. [Ali Ijaz Sheikh](https://github.com/ofrobots)
+1. [Anatoli Papirovski](https://github.com/apapirovski)
+1. [Anna Henningsen](https://github.com/addaleax)
+1. [Avraam Mavridis](https://github.com/AvraamMavridis)
+1. [Ben Michel](https://github.com/obensource)
+1. [Benedikt Meurer](https://github.com/bmeurer)
+1. [Benjamin Gruenbaum](https://github.com/benjamingr)
+1. [Benjamin Zaslavsky](https://github.com/Tiriel)
```

Comment:
Can I be added please? 
`[Beth Griggs](http://github.com/bethgriggs)`

Reply:
Done

Reply:
@BethGriggs you are added and @BridgeAR same! @BethGriggs if you are planning on attending the dinner please add yourself below as well!